### PR TITLE
Resolves console unicode issue #177

### PR DESCRIPTION
**Pull request by ramonsmits**

### CODE REVIEW

File: src/HashBus.Viewer/Program.cs [hunk 1]
```diff
@@ -3,12 +3,15 @@
...
     using System.Configuration;
     using System.Linq;
+    using System.Text;
     using System.Threading.Tasks;
 
     class Program
     {
         static Task Main()
         {
+            Console.OutputEncoding = Encoding.UTF8;
```
Comment (adamralph) [inline src/HashBus.Viewer/Program.cs]: I'd prefer to have this line right before the setting of `Console.Title` so we have all the console set up in one place.

**Comment by ramonsmits** [comment]
@adamralph Done

**Comment by ramonsmits** [comment]
@adamralph You can just squash merge this to master instead.

**Comment by ramonsmits** [comment]
@adamralph squashed and force pushed.

**Pull request merged**